tests/Admin/Extension/LockExtensionTest.php

```diff
@@ -240,7 +240,7 @@ private function configureAdmin(
         string $uniqid = '',
         ?Request $request = null
     ): void {
-        $this->admin->method('getUniqid')->willReturn($uniqid);
+        $this->admin->method('getUniqId')->willReturn($uniqid);
```

Comment:
the name of the property could be changed to

Reply:
Done